app/src/main.ts

```diff
@@ -3,9 +3,17 @@ import 'source-map-support/register';
 import fs from 'fs';
 import path from 'path';
 
-import { app, crashReporter, globalShortcut, BrowserWindow } from 'electron';
+import {
+  app,
+  crashReporter,
+  globalShortcut,
+  BrowserWindow,
+  dialog,
+} from 'electron';
 import electronDownload from 'electron-dl';
 
+import latestVersion from 'latest-version';
```

Comment:
Please refrain from adding extra dependencies to Nativefier. Packages get abandoned / insecure all the time, I only accept new dependencies if they're absolutely critical to new important functionality. Here, the new dep is unnecessary.

Also, using the logic you're suggesting would be very  noisy / frequently cumbersome: new Electron versions are published all the time, many without security issues for which it's not worthwhile to bother users. See https://github.com/jiahaog/nativefier/issues/556 for my suggestion, or feel free to suggest something better:

> *"Could be as simple as writing the build date, and checking at runtime if Date() > buildDate + a few months."*